# For ciwei100000 self-use to modify the jobserver number of
  LTO when building gcc

--- a/src/config/bootstrap-lto-noplugin.mk
+++ b/src/config/bootstrap-lto-noplugin.mk
@@ -1,8 +1,8 @@
 # This option enables LTO for stage2 and stage3 on
 # hosts without linker plugin support.
 
-STAGE2_CFLAGS += -flto=jobserver -frandom-seed=1 -ffat-lto-objects
-STAGE3_CFLAGS += -flto=jobserver -frandom-seed=1 -ffat-lto-objects
-STAGEprofile_CFLAGS += -flto=jobserver -frandom-seed=1
-STAGEtrain_CFLAGS += -flto=jobserver -frandom-seed=1
-STAGEfeedback_CFLAGS += -flto=jobserver -frandom-seed=1
+STAGE2_CFLAGS += -flto=8 -frandom-seed=1 -ffat-lto-objects
+STAGE3_CFLAGS += -flto=8 -frandom-seed=1 -ffat-lto-objects
+STAGEprofile_CFLAGS += -flto=8 -frandom-seed=1
+STAGEtrain_CFLAGS += -flto=8 -frandom-seed=1
+STAGEfeedback_CFLAGS += -flto=8 -frandom-seed=1

--- a/src/config/bootstrap-lto.mk
+++ b/src/config/bootstrap-lto.mk
@@ -1,10 +1,10 @@
 # This option enables LTO for stage2 and stage3 in slim mode
 
-STAGE2_CFLAGS += -flto=jobserver -frandom-seed=1
-STAGE3_CFLAGS += -flto=jobserver -frandom-seed=1
-STAGEprofile_CFLAGS += -flto=jobserver -frandom-seed=1
-STAGEtrain_CFLAGS += -flto=jobserver -frandom-seed=1
-STAGEfeedback_CFLAGS += -flto=jobserver -frandom-seed=1
+STAGE2_CFLAGS += -flto=8 -frandom-seed=1
+STAGE3_CFLAGS += -flto=8 -frandom-seed=1
+STAGEprofile_CFLAGS += -flto=8 -frandom-seed=1
+STAGEtrain_CFLAGS += -flto=8 -frandom-seed=1
+STAGEfeedback_CFLAGS += -flto=8 -frandom-seed=1
 
 # assumes the host supports the linker plugin
 LTO_AR = $$r/$(HOST_SUBDIR)/prev-gcc/gcc-ar$(exeext) -B$$r/$(HOST_SUBDIR)/prev-gcc/
